app/scripts/lib/metrics.js

```diff
@@ -90,6 +90,12 @@ define(function (require, exports, module) {
     return `flow.${viewName.replace(/^oauth\./, '')}.${eventName}`;
   }
 
+  function marshallUtmParam (utmParam) {
+    if (utmParam && utmParam !== 'none') {
```

Comment:
Couldn't find any docs about why we exclude the `none` value here. Is there other values we would want to exclude?

Reply:
It's a default value set in the constructor. It has no relationship to actual utm params set by the marketing team, hence we don't want to risk messing up their metrics by passing it on to them.